explcheck/src/explcheck-common.lua

```diff
@@ -0,0 +1,17 @@
+-- Transform a filename into a state object that can be passed to the processing steps.
```

Comment:
Change from `state` to `issues`. Remove `filename` and `content` from the object. Add methods for adding and ignoring issues. Pass everything else (expl3 code ranges) around explicitly.

Reply:
Fixed in commit 82e0dee2cd02c1d53633cddd42ab8fbd03f1d9f3 from this PR.